### PR TITLE
Shorter UA

### PR DESCRIPTION
**Pull request by mcardosos**
PTAL
@salameer @marstr @jhendrixMSFT

### CODE REVIEW

File: example.go [hunk 1]
```diff
@@ -79,7 +79,7 @@ func getEnvVarOrExit(varName string) string {
 }
 
 func createClients(subscriptionID string, authorizer *autorest.BearerAuthorizer) {
-	sampleUA := fmt.Sprintf("Azure-Samples/storage-go-resource-provider-getting-started/%s", utils.GetCommit())
+	sampleUA := fmt.Sprintf("sample/0001/%s", utils.GetCommit())
```
Comment (jhendrixMSFT) [inline example.go]: Why the change?

Comment (mcardosos) [inline example.go]: UA is getting too long

Comment (jhendrixMSFT) [inline example.go]: Did somebody complain, is it causing a real issue?  An example would be nice.

Comment (marstr) [inline example.go]: I agree with @jhendrixMSFT, there's some real value to having a readable user agent string, and the difference of payload is pretty much negligible. If we'd like unique identifiers for the samples, I would suggest we do so in an additive fashion.
